src/G2/Lib/Printers.hs

```diff
@@ -454,7 +458,7 @@ mkTypeHaskellPG pg (TyCon n _) | nameOcc n == "List"
                                , nameModule n == Just "GHC.Types" = "[]"
                                | otherwise = mkNameHaskell pg n
 mkTypeHaskellPG pg (TyApp t1 t2) = "(" <> mkTypeHaskellPG pg t1 <> " " <> mkTypeHaskellPG pg t2 <> ")"
-mkTypeHaskellPG pg (TyForAll i t) = "forall " <> mkIdHaskell pg i <> " . " <> mkTypeHaskellPG pg t
+mkTypeHaskellPG pg (TyForAll i t) = "forall " <> mkIdHaskell pg i <> " . (" <> mkTypeHaskellPG pg t <> " :: " <> mkTypeHaskellPG pg (typeOf t) <> ")"
```

Comment:
My fault for introducing it in the first place I think- but let's undo this change.  Should be clear what (typeOf t) is from the extra id info we print when aggressive mode is on.